tests/page_objects/onboarders_care_platform_page.yaml

```diff
@@ -1,3 +1,10 @@
+onboarders_platform_home:
+  page_title: //h3
+  call_queue_title: //h3[text()='Call Queue']
```

Comment:
this one too, where is it used? if there are new page objects that are not used, delete them.